apps/src/templates/ChallengeDialog.jsx

```diff
@@ -82,7 +82,7 @@ class ChallengeDialog extends React.Component {
           />
         </div>
         <div style={styles.content}>
-          <div style={styles.text}>{this.props.text}</div>
+          {this.props.text && <div style={styles.text}>{this.props.text}</div>}
```

Comment:
This change fixes a small UI bug that was bothering me. Because `styles.text` contains margin-bottom, when `this.props.text` was falsy, there was an awkward gap before `this.props.children`.

**Before:**
<img width="598" alt="Screen Shot 2021-09-22 at 2 30 10 PM" src="https://user-images.githubusercontent.com/9812299/134428800-75c21032-9750-40ef-bc4a-055c42094038.png">
**After:**
<img width="587" alt="Screen Shot 2021-09-22 at 2 58 56 PM" src="https://user-images.githubusercontent.com/9812299/134428560-8ff86b53-8a48-4c4f-8ffa-9c396f9238d0.png">